posthog/api/person.py

```diff
@@ -203,7 +203,7 @@ def list(self, request: request.Request, *args: Any, **kwargs: Any) -> response.
         elif not filter.limit:
             filter = filter.shallow_clone({LIMIT: DEFAULT_PAGE_LIMIT})
 
-        query, params = PersonQuery(filter, team.pk).get_query(paginate=True)
+        query, params = PersonQuery(filter, team.pk).get_query(paginate=True, filter_future_persons=True)
```

Comment:
nit: I think folks might be using the person API to get all the persons, so it could be nice if we provided an option to the request to specify to ignore this filter. Might also make sense to default to ignoring it if it's a csv request. Feel free to ignore.